### PR TITLE
Refactor(*): rewrite in flow, move to jest

### PR DESCRIPTION
**Pull request by btford**
- remove router and react wrapper for now
- fixes to how simulating success and error (in test, for combinators) works

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2016 Brian Ford
+Copyright (c) 2017 Brian Ford
```
Comment (balthazar) [inline LICENSE.md]: 8?

File: tasks.js [hunk 2]
```diff
@@ -1 +1,2 @@
-module.exports = require('./dist/src/tasks');
+// @flow
+module.exports = require('./dist/tasks');
```
Comment (balthazar) [inline tasks.js:2]: there's a trick, if you copy the `package.json` in the `dist` folder after the build, you can publish from there and avoid having these files polluting your root just to re-export bundled files

File: LICENSE.md (hunk 1 above)
Comment (btford) [reply LICENSE.md]: oh yeah

File: tasks.js (hunk 2 above)
Comment (btford) [reply tasks.js:2]: clever!

File: README.md [hunk 3]
```diff
@@ -40,21 +41,6 @@ const enhancers = [
 const store = createStore(reducer, initialState, compose(...enhancers))
 ```
 
-If you're using the redux-devtools extension, you might want to have readable actions types,
```
Comment (balthazar) [inline README.md:43]: We don't need this anymore then?

File: flow-typed/npm/ava_vx.x.x.js [hunk 4]
```diff
@@ -0,0 +1,227 @@
+// flow-typed signature: feea64f2baf0aea6ea952f8be6237b0f
```
Comment (balthazar) [inline flow-typed/npm/ava_vx.x.x.js]: If we switch to jest, do we still need this?

File: package.json [hunk 5]
```diff
@@ -1,64 +1,40 @@
...
   "main": "dist/src",
   "homepage": "https://github.com/btford/react-palm",
   "scripts": {
-    "build": "rm -rf dist && tsc",
-    "build-example": "npm run build && webpack dist/example/main.js dist/example/main.bundle.js",
-    "test": "npm run build && ava",
-    "flow": "flow && node ./scripts/perfect-coverage.js",
-    "watch": "node bin/watch.js"
+    "build":
+      "rm -rf dist && babel src --out-dir dist && flow-copy-source -v src/ dist",
```
Comment (balthazar) [inline package.json:9]: remove the newline here

File: package.json [hunk 6]
```diff
@@ -1,64 +1,40 @@
 {
   "name": "react-palm",
-  "version": "0.8.1",
+  "version": "1.0.0",
```
Comment (balthazar) [inline package.json:3]: baller

File: package.json (hunk 5 above)
Comment (balthazar) [reply package.json:9]: Should it copy the `package.json` too in order to publish from dist?

File: tasks.js (hunk 2 above)
Comment (balthazar) [reply tasks.js:2]: ah I guess you don't want to do it

File: package.json (hunk 5 above)
Comment (btford) [reply package.json:9]: I'd like to do that in the future, but I'd prefer to change as little as possible for now.

Comment (balthazar) [reply package.json:9]: 0k

**Pull request merged**